sdk/formrecognizer/azure-ai-formrecognizer/src/samples/java/com/azure/ai/formrecognizer/FormTrainingAsyncClientJavaDocCodeSnippets.java

```diff
@@ -19,17 +21,17 @@
      * Code snippet for {@link FormTrainingAsyncClient} initialization
      */
     public void formTrainingAsyncClientInInitialization() {
-        // BEGIN: com.azure.ai.formrecognizer.FormTrainingAsyncClient.initialization
+        // BEGIN: com.azure.ai.formrecognizer.training.FormTrainingAsyncClient.initialization
         FormTrainingAsyncClient formTrainingAsyncClient = new FormRecognizerClientBuilder().buildAsyncClient()
             .getFormTrainingAsyncClient();
-        // END: com.azure.ai.formrecognizer.FormTrainingAsyncClient.initialization
+        // END: com.azure.ai.formrecognizer.training.FormTrainingAsyncClient.initialization
     }
 
     /**
-     * Code snippet for {@link FormTrainingAsyncClient#beginTraining}
+     * Code snippet for {@link FormTrainingAsyncClient#beginTraining(String, boolean)}
      */
     public void beginTraining() {
-        // BEGIN: com.azure.ai.formrecognizer.FormTrainingAsyncClient.beginTraining#string-boolean
+        // BEGIN: com.azure.ai.formrecognizer.training.FormTrainingAsyncClient.beginTraining#string-boolean
         String trainingSetSource = "{training-set-SAS-URL}";
         boolean useLabelFile = true;
         formTrainingAsyncClient.beginTraining(trainingSetSource, useLabelFile).subscribe(
```

Comment:
probably better to change the code snippets to, making it trainingFilesUrl, useTrainingLabels